GDAPI/GDAPI.Tests/Objects/GeometryDash/LevelObjects/LevelObjectCollectionTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using GDAPI.Enumerations.GeometryDash;
+using GDAPI.Objects.GeometryDash.General;
+using GDAPI.Objects.GeometryDash.LevelObjects;
+using GDAPI.Objects.GeometryDash.LevelObjects.Triggers;
+using NUnit.Framework;
+using System.Linq;
+
+namespace GDAPI.Tests.Objects.GeometryDash.LevelObjects
+{
+    public class LevelObjectCollectionTests
+    {
+        [SetUp]
+        public void Setup()
```

Comment:
is there a reason on why we leave these blank?

Reply:
I think these can be actually omitted, so I'll make a new PR to remove them all.